deployment/chainloop/values.yaml

```diff
@@ -485,13 +485,26 @@ controlplane:
   ##       ...
   ##       -----END RSA PRIVATE KEY-----
   ## @param controlplane.keylessSigning.fileCA.keyPass The secret key pass
+  ## @param controlplane.keylessSigning.ejbcaCA.serverUrl The url of the EJBCA service (https://host/ejbca)
+  ## @param controlplane.keylessSigning.ejbcaCA.key PEM-encoded the private key for EJBCA cert authentication
+  ## @param controlplane.keylessSigning.ejbcaCA.cert PEM-encoded certificate for EJBCA cert authentication
+  ## @param controlplane.keylessSigning.ejbcaCA.certProfileName Name of the certificate profile to use in EJBCA
+  ## @param controlplane.keylessSigning.ejbcaCA.endEntityProfileName Name of the Entity Profile to use in EJBCA
+  ## @param controlplane.keylessSigning.ejbcaCA.caName Name of the CA issuer to use in EJBCA
   keylessSigning:
     enabled: false
     backend: fileCA
     fileCA:
       cert: ""
       key: ""
       keyPass: ""
+    ejbcaCA:
```

Comment:
we need to decide if we want disabled code to be commented out or with empty values. I lean towards commented out since not always you have a toggle like the one you have here with `backend`. But either way, we can move on :)

Reply:
I'm afraid keys must exist for the Readme generator to work properly, otherwise it would fail.

Reply:
Not really, you can  use the `@extra` annotation instead